tests/data/test_creation_from_file/remote.yaml

```diff
@@ -1,6 +1,6 @@
 chunk_size: 256
 local_device: null
-remote_url: "lm://localhost:65000"
+remote_url: "lm://localhost:65432"
```

Comment:
The port should be 65000. During the test, a `lmcache_server` process will be initialized at localhost:65000. Please refer to [conftest.py](https://github.com/LMCache/LMCache/blob/1f3ecc1d5949a7c8bfc973df1cbfaa56c1cdc0f0/tests/conftest.py#L34C1-L34C31)